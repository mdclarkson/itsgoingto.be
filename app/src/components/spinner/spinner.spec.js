import React from 'react'
import { render } from '@testing-library/react'
import Spinner from './spinner'

describe('(Component) Spinner', () => {
  describe('(Render)', () => {
    it('matches snapshot', () => {
      const { asFragment } = render(<Spinner />)
      expect(asFragment()).toMatchSnapshot()
    })
  })
})
