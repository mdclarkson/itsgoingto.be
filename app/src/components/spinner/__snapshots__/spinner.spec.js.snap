// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Spinner (Render) matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="spinner"
  >
    <span
      class="spinner_dot spinner_dot--first"
    />
    <span
      class="spinner_dot spinner_dot--second"
    />
    <span
      class="spinner_dot spinner_dot--third"
    />
  </div>
</DocumentFragment>
`;
